tests/Metrics/WikiMetricsTest.php

```diff
@@ -49,5 +49,32 @@ public function testDoesNotAddDuplicateRecordsWithOnlyDateChange()
             'date' => Carbon::today()->toDateString()
         ]);
     }
+
+    public function testAddRecordsWikiIsDeleted()
+    {
+        $wiki = Wiki::factory()->create([
+            'domain' => 'thisfake.wikibase.cloud'
+        ]);
+        //Insert an old metric value for a wiki
+        WikiDailyMetrics::create([
+            'id' => $wiki->id. '_'. Carbon::yesterday()->toDateString(),
+            'wiki_id' => $wiki->id,
+            'date' => Carbon::yesterday()->toDateString(),
+            'pages' => 0,
+            'is_deleted' => 0
+        ]);
+        //delete the wiki
+        $wiki->delete();
+        $wiki->save();
+
+        (new WikiMetrics())->saveMetrics($wiki);
+
+        //Assert No new record was created for today
```

Comment:
doesn't this assert that a new record *was* created for today?

Reply:
looks great other than this comment; please go ahead with it :)

Reply:
> doesn't this assert that a new record was created for today?

I think that's true. As the expected behaviour [seems to be](https://phabricator.wikimedia.org/T383421#10586021) that in case a wiki gets deleted one final row should get added where `is_deleted = 1`, I adjusted this test to first insert a row where exactly that happens "yesterday" but then no new row gets added "today"